odoo_install.sh

```diff
@@ -260,4 +361,7 @@ echo "Addons folder: $OE_USER/$OE_CONFIG/addons/"
 echo "Start Odoo service: sudo service $OE_CONFIG start"
 echo "Stop Odoo service: sudo service $OE_CONFIG stop"
 echo "Restart Odoo service: sudo service $OE_CONFIG restart"
+if [ $INSTALL_NGINX = "True" ]; then
+ echo "Done! The Nginx server is up and running. Configuration can be found at /etc/nginx/sites-available/$OE_USER"
```

Comment:
Should be `/etc/nginx/sites-enabled/$OE_USER ` (or `sites-available/odoo`) as we are creating symbolic link from `sites-available/odoo` to `sites-enabled/$OE_USER`

Reply:
True! I'd go with the variable though.